### DISCUSSION TITLE
stonedb-8.0-v1.0.1-beta

### DISCUSSION THREAD

**Discussion by Nliver** [stoneatom/stonedb#1849] [Announcements]
# 8.0-v1.0.1-beta
- Version Number: 8.0-v1.0.1-beta
- Release Date: 2023-06-01

## What's Changed
* workflow(git)：update the `.gitignore` by @Nliver in https://github.com/stoneatom/stonedb/pull/1789
* fix: github workflow exit with error.(#1800) by @chenshengjiang in https://github.com/stoneatom/stonedb/pull/1801
* fix(mtr): make sleep time longger to avoid no data select from db #1803 by @hustjieke in https://github.com/stoneatom/stonedb/pull/1804
* fix(core): shadow copy cause result dismatch #1736 by @hustjieke in https://github.com/stoneatom/stonedb/pull/1802
* fix(tianmu): backport EXTRACT() function error stoneatom#1691 by @Double0101 in https://github.com/stoneatom/stonedb/pull/1799
* fix: remove warnings of building release type.(#1679) by @chenshengjiang in https://github.com/stoneatom/stonedb/pull/1795
* feat(tianmu): support unsigned zerofill #1716 by @hustjieke in https://github.com/stoneatom/stonedb/pull/1813
* fix(tianmu): backport support union different datetime type #1743 by @Double0101 in https://github.com/stoneatom/stonedb/pull/1811
* fix(tianmu): disable primary key as tianmu engine did not support well by @hustjieke in https://github.com/stoneatom/stonedb/pull/1820
* fix(workflow): nightly build failed #1830 by @hustjieke in https://github.com/stoneatom/stonedb/pull/1837

**Full Changelog**: https://github.com/stoneatom/stonedb/compare/8.0-v1.0.1-alpha...8.0-v1.0.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/stoneatom/stonedb/releases/tag/8.0-v1.0.1-beta'>stonedb-8.0-v1.0.1-beta</a>.</em>

**Comment by zhenglin-charlie-li** [comment]
Hello,

Great to hear that StoneDB will 8.0 will release soon. 

I am an open source contributor in SQLancer community and I am trying to let SQLancer support the latest version of StoneDB. 

I am wondering how can I get the lateset version (8.0) on docker hub?

It seams that the latest version on Docker Hub is StoneDB-5.7-V1.0.3 but I need the lateset version (8.0).

**Reply by hustjieke** [reply]
Thanks very much, `SQLancer` is an awsome tool!
The newest version of StoneDB-8.0 has not been published into Docker hub yet, we are now doing test on it, the work will be finished in the next few days!

**Reply by zhenglin-charlie-li** [reply]
Got it. Thanks! Looking for the release of 8.0 verison.

**Reply by zhenglin-charlie-li** [reply]
Hi, @hustjieke. I have a question on how to run StoneDB by running the binary file? What commands do we need to execute? 

It seams that there is no `reinstall.sh` file under the folder of `/stonedb57/install`? But the docs says that we need to run this file: https://stonedb.io/docs/getting-started/quick-deployment/quick-deployment-57/#43-automatically-install-stonedb
